utils/acpica/Makefile

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=acpica
+PKG_VERSION:=20170531
+PKG_RELEASE:=1
+
+PKG_BASENAME:=$(PKG_NAME)-unix
+PKG_SOURCE:=$(PKG_BASENAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://acpica.org/sites/$(PKG_NAME)/files/$(PKG_SOURCE)
+PKG_MD5SUM:=cf00eec367b996219f640756895f414f
+PKG_HASH:=50155778cd1633dfca3443b8f8fd1ccc30e70e55ddece4d3c4fceafbbf1ab0e8
+PKG_MAINTAINER:=Philip Prindeville <philipp@redfish-solutions.com>
```

Comment:
PKG_HASH is also already supported in openwrt now so that PKG_MD5SUM can be omitted here.